src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/IPackageMetadata.cs

```diff
@@ -48,5 +49,7 @@ public interface IPackageMetadata
         /// Returns sets of Content Files specified in the manifest.
         /// </summary>
         ICollection<ManifestContentFiles> ContentFiles { get; }
+
+        IReadOnlyList<PackageType> PackageTypes { get; }
```

Comment:
I considered this, but I wanted simple counting and indexing. On the other hand, this inconsistency is weird. Perhaps consistency is more important in this case. I'll see what it looks like to switch over all of these to `IEnumerable<T>`.